README.md

```diff
@@ -104,6 +95,27 @@ It means:
 - alice can read data1
 - bob can write data2
 
+We also support multi-line mode by appending '\\'  in the end:  
+
+```ini
+# Matchers
+[matchers]
+m = r.sub == p.sub && r.obj == p.obj \ 
+  && r.act == p.act
+```
+
+Further more, if you are using ABAC,  you can try operator `in` like following:
+
+```ini
+# Matchers
+[matchers]
+m = r.obj == p.obj && r.act == p.act || r.obj in ('data2', 'data3')
+```
+
+But you **SHOULD ** make sure that the length of the array is **MORE **than **1**,  otherwise there will cause a panic.
```

Comment:
Should be **SHOULD** or **SHOULD **? The same about MORE.

Also there's an English grammar issue in ``there will cause a panic.``

Reply:
Alright, there are something wrong with my MarkDown editor